src/login/index.html

```diff
@@ -62,19 +62,26 @@ <h1 localization-key="loginTitle"></h1>
                 placeholder="" />
               <a href="#"
                 localization-key="retrievePasswordLink"></a>
+              <label for="mfa_token"
+                localization-key="mfaLabel"></label>
+              <input id="mfa_token"
+                name="mfa_token"
```

Comment:
Just a minor nitpick, but the other IDs and names are named using hyphens so this could also be **mfa-token**.

Reply:
No problem, I'll change it to hyphens :+1: